lib/trino-parquet/src/main/java/io/trino/parquet/writer/valuewriter/FixedLenByteArrayLongDecimalValueWriter.java

```diff
@@ -37,7 +37,7 @@ public FixedLenByteArrayLongDecimalValueWriter(ValuesWriter valuesWriter, Type t
         super(parquetType, valuesWriter);
         this.decimalType = (DecimalType) requireNonNull(type, "type is null");
         checkArgument(!this.decimalType.isShort(), "type is not a long decimal");
-        checkArgument(parquetType.getTypeLength() <= Int128.SIZE, "Type length %s exceeded 16 bytes", parquetType.getTypeLength());
+        checkArgument(parquetType.getTypeLength() <= Int128.SIZE, "Type length %s exceeds 16 bytes", parquetType.getTypeLength());
```

Comment:
nit: you use `Int128.BYTES` for the check, but hardcode `16` in the message. looks slightly weird

btw is the wording intentionally different than in the other class?